internal/backend/db/dbgorm/triggers.go

```diff
@@ -14,24 +14,33 @@ import (
 
 func triggerToRecord(ctx context.Context, tx *tx, trigger sdktypes.Trigger) (*scheme.Trigger, error) {
 	connID := trigger.ConnectionID()
+	var conn sdktypes.Connection
+	var err error
+	var projID sdktypes.ProjectID
 
-	conn, err := tx.GetConnection(ctx, connID)
-	if err != nil {
-		return nil, fmt.Errorf("get trigger connection: %w", err)
+	if connID.IsValid() {
```

Comment:
Shouldn't there be any mention of schedule as being required, if `!connID.IsValid()`?

Reply:
the only validity check on db layer is foreign keys used. Other constrains are checked elsewhere, either in proto or in sdk layer. For trigger plz see `sdktypes.trigger`